core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -970,10 +981,13 @@ trait EntityComponent {
                 val allConflictRefs = softConflicts.flatMap(_.path)
                 val entitiesToCopy = allEntityRefs diff allConflictRefs
 
+                val entityRefsToCopy = entitiesToCopy.map(e => AttributeEntityReference(e.entityType, e.entityName)).toSet
+                val entitiesToCopyChunks = entityRefsToCopy.grouped(batchSize)
```

Comment:
`entitiesToCopy` already contains `AttributeEntityReference`s, no need for that first map:
```suggestion
                val entitiesToCopyChunks = entitiesToCopy.toSet.grouped(batchSize)
```

Reply:
Good catch.  I know it's our Scala idiom but I miss seeing the type info.

Reply:
I prefer seeing type info too, I am not attached at all to omitting it all the time.